grpc/src/main/java/io/stargate/grpc/impl/GrpcImpl.java

```diff
@@ -59,6 +60,7 @@ public GrpcImpl(
             .intercept(new AuthenticationInterceptor(authenticationService))
             .intercept(new RemoteAddressInterceptor())
             .intercept(new HeadersInterceptor())
+            .intercept(new MetricCollectingServerInterceptor(metrics.getMeterRegistry()))
```

Comment:
So we are going with the default stuff from Micrometer right? Any chance we can add a small INT test here, just to confirm it works when being hit?

Reply:
Correct. I'll add some tests